src/lib/components/Tabs/index.js

```diff
@@ -117,6 +147,10 @@ export class Tabs extends BaseElement {
     `;
   }
 
+  loadmorePanel() {
+    return html`<div class="web-tabs__panel" role="tabpanel" hidden></div>`;
+  }
```

Comment:
I think that's fine to inline 🙂 doesn't take any arguments and is just one line.